### ISSUE TITLE
Re-think how we measure "good" defense

### ISSUE THREAD

**Issue by Faraday-dot-py** [Patribots4738/ScoutingWebsite#5]
Some ideas we've had are:
- Removing it entirely and having 2 people scout for just defense separately (only works with 14+ people)
- Mix between timer and rating
- "Did they defend" boolean and "defense comments"
- How many fouls they scored
- Pin time

**Comment by GalexY727** [comment]
I like the pin timer, and what if instead of the defense timer encapsulating all of their defense time, what if the defense timer would only be toggled if they are doing *good* defense? for example, if 8006 soft blocked out 4738 for 5 seconds in the match, then the defense timer would only be at 5, even though 8006 was on defense the whole match.

**Comment by RandomPro8** [comment]
We still need to establish a standard for what "good" defense is

**Comment by GalexY727** [comment]
Right, and contrary to what i was just suggesting, A robot that did one match of perfect defense cannot be put below a robot that did mediocre defense for more time

**Comment by GalexY727** [comment]
Maybe make an average of how many blocks a robot made in a match for all matches they attempted defense?

**Comment by GalexY727** [comment]
i can help with that
